pkg/porter/lifecycle_test.go

```diff
@@ -668,3 +669,53 @@ func Test_ensureVPrefix_idempotent(t *testing.T) {
 		})
 	}
 }
+
+// ensure no v is added if specifying :latest tag
+func Test_ensureVPrefix_latest(t *testing.T) {
+
+	latestRef := "example/porter-hello:latest"
+
+	t.Run(":latest tag", func(t *testing.T) {
+		opts := BundleReferenceOptions{
+			installationOptions: installationOptions{},
+			BundlePullOptions: BundlePullOptions{
+				Reference:        latestRef,
+				_ref:             nil,
+				InsecureRegistry: false,
+				Force:            false,
+			},
+			bundleRef: nil,
+		}
+
+		err := ensureVPrefix(&opts, os.Stdout)
```

Comment:
```suggestion
		err := ensureVPrefix(&opts, io.Discard)
```
So we don't have all the noise from the test because we are not asserting against the output.